clients/java/signalr/src/main/java/WebSocketTransport.java

```diff
@@ -37,8 +39,10 @@ else if(url.startsWith(HTTP)){
 
     @Override
     public void start() throws InterruptedException {
+        logger.log(LogLevel.Debug, "Starting Websocket connection");
```

Comment:
Again, take a quick look at the WebSocket transport in C# and make sure the levels and messages align where they should